proto/stream_plan.proto

```diff
@@ -142,8 +142,8 @@ message SinkDesc {
   string name = 2;
   string definition = 3;
   repeated plan_common.ColumnDesc columns = 4;
-  repeated common.ColumnOrder pk = 5;
-  repeated uint32 stream_key = 6;
+  repeated common.ColumnOrder plan_pk = 5;
```

Comment:
Will we use the plan_pk anywhere?

Reply:
And do we enforce any constraint on the relation between `plan_pk` and `downstream_pk` in this PR?

Reply:
> And do we enforce any constraint on the relation between `plan_pk` and `downstream_pk` in this PR?

No. `downstream_pk` is specified by the user, and the user can specify anything he or she wants, as long as the columns are valid.